docs/specification.md

```diff
@@ -34,14 +34,13 @@ An existing separate FA2 contract is required and used for governance.
 # Configuration
 
 BaseDAO is a concrete smart contract, but also a framework to implement various DAOs.
```

Comment:
I'd say that `baseDAO` is no longer a concrete smart contract, given that only its variants can be built.

Reply:
Can you please suggest a good way to describe it at its current state?

Reply:
Well, I would simply remove the "concrete smart contract" part and leave the fact that it is a framework.

I'd suggest:
```suggestion
BaseDAO is a framework to implement various DAO smart contracts.
```